src/Generator.h

```diff
@@ -3478,6 +3484,9 @@ class GeneratorBase : public NamesInterface, public AbstractGenerator {
 
     const size_t size;
 
+    // Flag to enable serialization (and bypass asserts for unbound target)
```

Comment:
per previous comments, this seems no longer necessary or desirable

Reply:
Fixed!